src/class-list-table.php

```diff
@@ -29,19 +29,25 @@ public function column_default( $item, $column_name ) : string {
 				return isset( $item['content_type'] ) ? ucwords( implode( ', ', $item['content_type'] ) ) : __( 'Post, Page', 'wp-decoupled-preview' );
 			case 'actions':
 				return sprintf(
-					'<a href="%s">%s</a> | <a href="%s&TB_iframe=true&width=600&height=300" class="thickbox">Test</a>',
+					'<a href="%s">%s</a> | <a href="%s&TB_iframe=true&width=600&height=300" class="thickbox">%s</a> | <a href="%s&TB_iframe=true&width=600&height=300" class="thickbox">%s</a>',
```

Comment:
Now that we're displaying more info here, I think we could use more height. Could you make the height larger? Maybe 500 or 600?